docker/docker-compose.yml

```diff
@@ -210,6 +214,26 @@ services:
     profiles:
       - ray
 
+  ollama:
+    image: ollama/ollama:0.3.4
+    ports:
+      - ${OLLAMA_EXPOSED:-19290}:11434
+    environment:
+      - OLLAMA_KEEP_ALIVE=24h
+    tty: true
+    restart: unless-stopped
+    volumes:
+      - ./ollama_cache:/root/.ollama
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              device_ids: ["0"]
+              capabilities: [gpu]
+    networks:
```

Comment:
I would add a docker profile "ollama" similar to the "ray" profile to control whether the container should be started or not